src/utils/react-live-utils.js

```diff
@@ -0,0 +1,83 @@
+
+function transformCode (code) {
+  return `() => {
+    ${code}
+    return null
+  }`
+}
+
+function log (fn) {
+  const _log = console.log
+  const _error = console.error
+  const _warning = console.warning
+  const logProxy = {}
+  logProxy.error = function (...args) {
```

Comment:
new line

Reply:
Additionally, you might want to use `lodash.wrap` for all the logProxy stuff. This will reduce a lot of code here.